src/ert/storage/local_experiment.py

```diff
@@ -200,4 +203,7 @@ def write_simulation_arguments(
         with open(
             self.mount_point / self._simulation_arguments_file, "w", encoding="utf-8"
         ) as f:
-            json.dump(dataclasses.asdict(info), f)
+            try:
+                json.dump(dataclasses.asdict(info), f)
+            except TypeError:
+                logger.error(f"Failed to serialize: {info}")
```

Comment:
This depends on this on `info `being serializable to text, but I guess this is an OK assumption?

Reply:
Also maybe not error?

Reply:
If we can continue I'll argue it's a warning

Reply:
It was an error that caused us to notice it though, but we will work on a proper fix for this once we get some more data, with the goal of removing it altogether.